### ISSUE TITLE
wbw Recitation Error - 2:181

### ISSUE THREAD

**Issue by thecashewtrader** [quran/quran.com-api#366]
The word by word recitation for 2:181 gets misaligned after word 3
https://quran.com/2/181
https://audio.qurancdn.com/wbw/002_181_003.mp3

**Comment by abdulraoufatia** [comment]
Hi, many thanks for your request. 

Regarding the word by word recitation - the word is correctly pronounced. However, it is recited without Tajweed in this incidence which produces the attached form you had provided. In addition to this, pronunciations of recitations are impacted by the style of reading. 

The style of Reading is further influenced by:
- your mother tongue and how it pronounces certain letters, i.e. in East Asia, د - ذ it is oftenpronounced as a "z", however in the middle east and northern africa it is often pronounced as "D and tha"
- Style of Reader you follow


**Styles of Recitation** 
There are many styles of recitation, the majority of scholars of recitation and usool are of the view that the seven modes of recitation are mutawaatir going back to the Prophet (blessings and peace of Allah be upon him). Some of them differed concerning that, such as Abu Shaamah – according to one of his views – and at-Tawfi and ash-Shawkaani. However, the correct view concerning that is the view of the majority; that is the right view which must be adopted.

Read more here: https://islamqa.info/en/answers/178120/the-seven-modes-of-recitation-are-mutawaatir-and-it-is-not-permissible-to-cast-aspersions-on-them 

Further more:
You should note, that the Qur’an was revealed in one style at the beginning, but the Messenger of Allaah (peace and blessings of Allaah be upon him) kept asking Jibreel until he taught him seven styles, all of which were complete. The evidence for that is the hadeeth of Ibn ‘Abbaas who narrated that the Prophet (peace and blessings of Allaah be upon him) said: “Jibreel taught me one style and I reviewed it until he taught me more, and I kept asking him for more and he gave me more until finally there were seven styles.”

(narrated by al-Bukhaari, 3047; Muslim, 819) 

The seven readers or reciters were: 

1. Naafi’ al-Madani
2. Ibn Katheer al-Makki
3. Aasim al-Kufi
4. Hamzah al-Zayaat al-Kufi
5. Al-Kisaa’i al-Kufi
6. Abu ‘Amr ibn al-‘Ala’ al-Basri
7. ‘Abd-Allaah ibn ‘Aamir al-Shaami

The ones who have the strongest isnaad in recitation are Naafi’ and ‘Aasim. 

The most eloquent are Abu ‘Amr and al-Kisaa’i. 

Warsh and Qaaloon narrated from Naafi’.

Hafs and Shu’bah narrated from ‘Aasim. 

Allah Knows Best

**Conclusion**
In summary, the attached recording of 3rd word ( بَعْدَ مَا ) is pronounced without tajweed. In addition to this, the recitation uses its own style of reciting (he is adding two words together ( بَعْدَ + مَا ) that has the same meaning )= "After + what" English Direct Translation 

I hope this makes more sense. I do advise to conduct your own research and gain your personal understanding. 

Jazak Allah Khairan and Allah Knows Best

**Comment by abdulraoufatia** [comment]
Hi, apologies for closing, just letting you know i have tried it through a different type of devices
pc - Not working correctly
Android - Not working correctly

I have re-opened the issue, apologies for the convenience caused.

Thank you for your support.

**Comment by thecashewtrader** [comment]
جزاك اللهُ خيرfor the reply.
 
 1.
 My issue with the recitation was not with the tajweed, but that it would be misaligned.
 So when u get word 8, it would play word 9.
 A possible fix would be to have word 3 and word 4 as the same (I remember seeing this being used in the api somewhere else)
 
 2. I wanted to confirm if these sounds are fine to use in my own projects (my project is GPL V3 licensed at https://gitlab.com/thecashewtrader/simple-quran, website https://thecashewtrader.gitlab.io/simple-quran)
